docs/docs/user-setup.md

```diff
@@ -88,34 +88,40 @@ powershell -Command "iex ((New-Object System.Net.WebClient).DownloadString('http
 </TabItem>
 </Tabs>
 
-### MCU Board Selection
+### Keyboard Selection
 
-When prompted, enter the number for the corresponding MCU board you would like to target:
+:::note
+If you are building firmware for a new keyboard or shield that is not included in the built-in
+list of keyboards, you can choose any keyboard or shield from the list that is similar to yours to generate the repository,
+and edit / add necessary files according to the [guide for adding new keyboard shield](development/new-shield.md).
```

Comment:
It might be good to qualify that the guide is for shields and not for integrated controllers as such:
```suggestion
and edit / add necessary files. You can follow the [new shield guide](development/new-shield.md) if you are adding support for a composite keyboard.
```